crates/milli/src/vector/mod.rs

```diff
@@ -442,27 +441,27 @@ impl ArroyWrapper {
         item: ItemId,
         limit: usize,
         filter: Option<&RoaringBitmap>,
-    ) -> Result<Vec<(ItemId, f32)>, arroy::Error> {
+    ) -> Result<Vec<(ItemId, f32)>, hannoy::Error> {
         if self.quantized {
             self._nns_by_item(rtxn, self.quantized_db(), item, limit, filter)
         } else {
             self._nns_by_item(rtxn, self.angular_db(), item, limit, filter)
         }
     }
 
-    fn _nns_by_item<D: arroy::Distance>(
+    fn _nns_by_item<D: hannoy::Distance>(
         &self,
         rtxn: &RoTxn,
-        db: arroy::Database<D>,
+        db: hannoy::Database<D>,
         item: ItemId,
         limit: usize,
         filter: Option<&RoaringBitmap>,
-    ) -> Result<Vec<(ItemId, f32)>, arroy::Error> {
+    ) -> Result<Vec<(ItemId, f32)>, hannoy::Error> {
         let mut results = Vec::new();
 
         for reader in self.readers(rtxn, db) {
             let reader = reader?;
-            let mut searcher = reader.nns(limit);
+            let mut searcher = reader.nns(limit, limit * 2); // TODO find better ef
```

Comment:
@nnethercott What would be the best default value we could put, and should we base it on the limit?